Day-6-7-8/src/components/SearchBar.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+
+const SearchBar = ({ searchMembers, users, fetchUserData }) => {
+  const [searchText, setSearchText] = useState("");
+
+  const filterMembers = (searchText) => {
+    return users.filter((mem) =>
+      mem.login.toLocaleLowerCase().includes(searchText.toLocaleLowerCase())
+    );
+    // return users[0];
+  };
+  const formSubmitHandler = (e) => {
+    e.preventDefault();
+    console.log(searchText);
+    if (searchText) {
+      const restList = filterMembers(searchText);
+      searchMembers(restList);
+    } else {
+      fetchUserData();
+    }
+    //
+
+    // searchMembers(restList);
+  };
+
+  const searchMember = (e) => {
+    setSearchText(e.target.value);
+  };
+
+  return (
+    <div className="search-bar">
+      <form onSubmit={formSubmitHandler}>
```

Comment:
can name handlers as `handle<something>`
React docs suggest that - `By convention, it is common to name event handlers as handle followed by the event name. You’ll often see onClick={handleClick}, onMouseEnter={handleMouseEnter}, and so on.`
refer: https://beta.reactjs.org/learn/responding-to-events#adding-event-handlers